kie-server-parent/kie-server-controller/kie-server-controller-standalone-war/src/main/shared-ee6-ee7-resources/META-INF/services/org.kie.server.controller.api.storage.KieServerTemplateStorage

```diff
@@ -0,0 +1 @@
+org.kie.server.controller.impl.storage.FileBasedKieServerTemplateStorage
```

Comment:
It was showing up in the ee6 and ee7 war files. It should now show up in all war files generated.